connection.go

```diff
@@ -613,10 +619,14 @@ func (c *Connection) sendQueued(bareAcks bool) (int, error) {
 
 // Send a packet of stream frames, plus whatever acks fit.
 func (c *Connection) sendCombinedPacket(pt uint8, frames []frame, acks ackRanges) (int, error) {
-	left := c.mtu
 	asent := int(0)
 	var err error
 
+	left := c.mtu
+	aead := c.determineAead(pt)
+	left -= aead.Overhead()
+	left -= kLongHeaderLength //TODO make this check if we are using a long or short header
```

Comment:
Why don't you combine these in the first definition of left.

Also, please use TODO(name). If you think you're going to do it, mark it as yourself, otherwise me.

Reply:
Sure, les goûts et les couleurs ne se discutent pas :)